index.html

```diff
@@ -3,7 +3,435 @@
+          </div>
+          <button class="subscribe" type="button" aria-label="icon">Subscribe</button>
+        </div>
+
+        <div>
+          <p class="show-more-text">
+            How do you contribute to open source? In this video, I'll describe the easiest way to
+            make a simple contribution to an open source project -- without using the command line,
+            a text editor, "forking a repo", or even touching Git outside of Github. I'll also
+            describe what kinds of contributions open source software projects are typically looking
+            for. As an example, I use submitting a student solution to a web development exercise
+            from The Odin Project, a free online curriculum for learning web development with Ruby
+            on Rails. You can find more information about that project at
+          </p>
+          <a href="#" target="_blank" aria-label="odin project website"
+            >http://theodinproject.com</a
+          >
+        </div>
+        <p class="show-more">Show more</p>
+
+        <section class="comment-section">
+          <header class="comment-head">
+            <p class="comment">2,493 Comments</p>
+            <p class="sort">SORT BY</p>
+          </header>
+          <div class="comments">
+            <img src="./image/abimbolayoutube.jpg" class="comment-img" alt="Avatar image" />
+            <p class="add-comment">Commenting publicly as Abimbola Buari</p>
+          </div>
+          <div class="other-comment">
+            <img class="comment-image" alt="avatar image" src="./image/commentyoutube.jpg" />
+            <div class="comment-block">
+              <h3>Nnechi<span> 4 year ago</span></h3>
+              <p>
+                I love this topic,Have been wanting to contribute to open source project for a long
+                to]ime but dont know how to go about it
+              </p>
+              <div class="comment-reply">
+                <p class="thousands">3K</p>
+                <p class="reply">REPLY</p>
+                <p>View 32 replies</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="other-comment">
+            <img class="comment-image" src="./image/comment2youtube.jpg" alt="Avatar image" />
+            <div class="comment-block">
+              <h3>Adeyemi <span> 4 year ago</span></h3>
+              <p>
+                Thumbs up!..really interesting topic,am an aspiring web developer that want to
+                contribute to open source.
+              </p>
+              <div class="comment-reply">
+                <i class="fas fa-thumbs-up" aria-hidden="true"></i>
+                <span class="thousands">8K</span>
+                <span class="reply">REPLY</span>
+              </div>
+              <div class="comment-replies">
+                <i class="fas fa-sort-down" aria-hidden="true"></i>
+                <p>View 5 replies</p>
+              </div>
+            </div>
+          </div>
```

Comment:
Let's use the `article` element for all section like this instead of `div` element since they seem to have a special meeting and structure on the page.